tools/e2etesting/pipeline_orchestrated.yml

```diff
@@ -1,8 +1,27 @@
 name: $(Date:yyyyMMdd)$(Rev:rr)
-
 trigger: none
 pr: none
 
+schedules:
+- cron: "0 10 * * *"
+  displayName: Nightly build of main branch.
+  branches:
+    include:
+    - main
+  always: true
+
+# Trigger Orchestrated E2E Pipeline post completion of build pipeline.
+resources:
+  pipelines:
+  - pipeline: E2E Orchestrated run after CI build
+    source: Custom\Azure_IOT\Industrial\Components\Azure.Industrial-IoT
```

Comment:
Should we have a variable for this and configure it to match our setup?

Reply:
Can you elaborate more? Are you suggesting to use variable to define source ?

Reply:
Taking care of these changes. Source will come from "Variables"

Reply:
Yes, the path should be configurable when the pipeline is set up in our devops instance.  That allows other users to reuse the templates and acripts and run them tests in their devops instances.

Reply:
As per this DevOps article, Trigger blocks cannot use variables. So I have to revert this change.
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#triggers